### ISSUE TITLE
onboard @EdOverflow

### ISSUE THREAD

**Issue by EdOverflow** [gratipay/inside.gratipay.com#952]
Hi Gratipay team,

My name is Ed and I am a web developer and a security researcher.

I hope by joining the blue team that I will be able to:

- Help protect Gratipay's customers from security vulnerabilities.
- Investigate, diagnose and fix security issues on your [HackerOne](https://hackerone.com/gratipay) program.

**Comment by whit537** [comment]
Huzzah! So ... you're off and running making PRs. May I add you to our GitHub org so you can make them on branches in the main repo here?

**Comment by EdOverflow** [comment]
OK, sure. 😃

**Comment by whit537** [comment]
Yay! Invited! 💃

**Comment by EdOverflow** [comment]
Thanks. :)

**Comment by whit537** [comment]
May I also add you to our [weekly money distribution](http://inside.gratipay.com/howto/take-money)? :-)

**Comment by EdOverflow** [comment]
I'll need to get that set up and will let you know when I am ready.

**Comment by whit537** [comment]
@EdOverflow May I add you as a "coordinator" in Transifex for French and German? That will enable you to self-review your own translations and review the translations of others. Do to limitations of Transifex, all translations must be marked as reviewed before we can deploy them.

**Comment by EdOverflow** [comment]
OK, great!

**Comment by whit537** [comment]
Done!

**Comment by whit537** [comment]
@EdOverflow Got your @gratipay/security team request and approved it. 👍

**Comment by EdOverflow** [comment]
Thank you. Would it be possible to add [me](https://hackerone.com/edio) to your HackerOne program?

**Comment by whit537** [comment]
> Would it be possible to add me to your HackerOne program?

Invite sent!

> could you please allow issue submissions on @gratipay/security?

I think you mean on the https://github.com/gratipay/security repo, ya? Why do you want to use that instead of HackerOne (for private things) or the inside.gratipay.com repo (for public things)?

**Comment by EdOverflow** [comment]
Never mind I just realised that Gratipay members report vulnerabilities on HackerOne too.

**Comment by EdOverflow** [comment]
Hi @whit537,

I have set up my Gratipay account: https://gratipay.com/~EdOverflow/. Feel free to add me to your [weekly money distribution](http://inside.gratipay.com/howto/take-money) when you find time.

**Comment by whit537** [comment]
@EdOverflow We're close! The last thing we need is a [national identity](https://gratipay.com/about/me/identities/) on file for you. We ask for this so that we can handle taxes appropriately. Are you willing to share that info with us? I invite you to review our [security practices](http://inside.gratipay.com/howto/keep-secrets#toc-2) around storing your PII, as well as our [audit](https://github.com/gratipay/gratipay.com/pull/3998) of the symmetric encryption library we're using.

**Comment by whit537** [comment]
Also, do you ever use Slack? I invite you to [join us there](http://inside.gratipay.com/appendices/chat) for real-time chat if you're interested. :)

**Comment by whit537** [comment]
[e.g.](http://gratipay.slackarchive.io/gratipay/-/1482426402.00115/1482876999.001272/1482876999001272/) ;-)

**Comment by whit537** [comment]
Following up from https://github.com/gratipay/gratipay.com/issues/4263 and https://github.com/gratipay/gratipay.com/issues/4262 ... have you used [GitHub projects](https://github.com/orgs/gratipay/projects/) at all? We're finding those to be helpful for organizing larger-scale projects that transcend a single ticket or repo. There are "✈️  Flight Deck  ✈️" labels in [`gratipay.com`](https://github.com/gratipay/gratipay.com/labels/%E2%9C%88%EF%B8%8F%20%20Flight%20Deck%20%20%E2%9C%88%EF%B8%8F) and [`inside.gratipay.com`](https://github.com/gratipay/inside.gratipay.com/labels/%E2%9C%88%EF%B8%8F%20%20Flight%20Deck%20%20%E2%9C%88%EF%B8%8F) that you can apply to tickets such as the ones you created, which can provide a place to coordinate work happening across all of the tickets in a given project.

We are also finding GitHub projects helpful for keeping track of what each of us is personally paying attention to. Those are what the "Radar" and "Queue" projects are for. I invite you to create a radar/queue project for yourself where you can publish for the rest of us what you are working on.

P.S. In general, please use organization-scope projects instead of repo-specific projects, so that the rest of us only have one place to look to discover projects.

**Comment by whit537** [comment]
We use LastPass to manage [some passwords](http://inside.gratipay.com/howto/keep-secrets), now including report-uri.io. Would you like access to our report-uri.io account? What email address should I use to invite you to LastPass? (You can tell me privately on chad@gratipay.com if you're not comfortable sharing here.)

**Comment by whit537** [comment]
@EdOverflow I received your email and sent invites for both Slack and LastPass.

**Comment by EdOverflow** [comment]
Thanks @whit537! You are always on the ball.

**Comment by whit537** [comment]
Except when I'm not! ☺️ 🏀

**Comment by whit537** [comment]
@EdOverflow I am seeing a lot of ambition from you:

- "[Full performance report](https://github.com/gratipay/gratipay.com/issues/4262)"
- "[Full security report](https://github.com/gratipay/gratipay.com/issues/4263)"
- "[complete redesign](https://github.com/gratipay/gratipay.com/pull/4268#issuecomment-269859500)"

That's great! It's also a _lot_ to bite off and chew. :-)

Can I help you think through your priorities and how to pace yourself and what your expectations are and how you plan to get all of these ambitious projects done? I don't want to see you get swamped and frustrated because progress is too slow. How do you see these three projects relating to one another?

**Comment by EdOverflow** [comment]
These are my goals for 2017. I have a clear plan with everything in order of priority. Obviously there is no way I can do all of this on my own, but I hope by communicating with the team, we can organise this together and get everything done.

| Project Name            | Priority Score | Description                                                                                   | Notes                                                                                                                                                                                                  |
|-------------------------|----------------|-----------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Full Security Report    | 10             | Set clear guidelines for developers and ensure there is a clear security process at Gratipay. On top of that, I believe Gratipay needs to have a threat model. | I will make this my biggest priority and will work on this publicly in small steps. I might need input from fellow Gratipay members.                                                                   |
| Full Performance Report | 7              | Set clear guidelines for developers when coding, in order to improve Gratipay's performance. | This is the easiest of all my goals, since I have already done a lot of testing.                                                                                                                       |
| Complete Redesign       | 4              | Redesign Gratipay's website. Create a landing page and a style guide.                         | Although this is linked to the performance report, this aim is actually a little more complex. I hope to organise a team of designers + developers and follow the "Performace Report" guidelines. This is where all the points raised in the "Performance Report" become reality. |

Thank you for being concerned @whit537. I wish you and the Gratipay team a happy New Year. 🎉🎉🎉

On a side note, I want to introduce the team to more tools by Scott Helme ([report-uri.io](https://report-uri.io/), [securityheaders.io](https://securityheaders.io/), [hardenize.com](https://www.hardenize.com)) and then document the process of how we used them and how we improved our platform's security (This does not directly belong to the "Security Report")

**Comment by whit537** [comment]
Awesome, looking forward to working together in 2017! 💃 

[!m](http://motivate.im/) @EdOverflow

**Comment by EdOverflow** [comment]
> @whit537: Would you like access to our report-uri.io account?

Yes, please.

**Comment by whit537** [comment]
I find a password for report-uri.io in 1password, but it does not appear to work. I issued a password reset and received the mail, but when I click the button in the email I just get the [asset png](https://report-uri.io/assets/email/reset-password-button.png)(?). Not sure if that's a bug in Report URI or Freshdesk.

![screen shot 2017-07-10 at 12 40 43 pm](https://user-images.githubusercontent.com/134455/28036405-03512584-656d-11e7-8ce3-fa5ad7788801.png)

**Comment by whit537** [comment]
I'm considering https://github.com/gratipay/gratipay.com/issues/4526 instead.

**Comment by EdOverflow** [comment]
@whit537 Scott Helme believes that this is a Zendesk issue.

**Comment by whit537** [comment]
Could be (though it's Freshdesk, not Zendesk).

I've sent a Sentry invite to the email you have on file on Gratipay.

**Comment by EdOverflow** [comment]
> Could be (though it's Freshdesk, not Zendesk).

:man_facepalming: 

> I've sent a Sentry invite to the email you have on file on Gratipay.

Thank you and accepted.

**Comment by whit537** [comment]
Per [slack](https://gratipay.slackarchive.io/gratipay/page-100/ts-1500618009552265) I've granted @EdOverflow admin perms on HackerOne.

> This group has Admin and Program permissions

**Comment by whit537** [comment]
Per [slack](https://gratipay.slackarchive.io/gratipay/page-100/ts-1502100977285922) (scroll down?) I've granted @EdOverflow agent permissions on Freshdesk to respond to security@ emails.

![](https://user-images.githubusercontent.com/134455/29041247-42cba648-7b80-11e7-9244-1c933a431669.png)
